src/loops/simple_programs_with_repitition/easy/CountItUp.java

```diff
@@ -0,0 +1,25 @@
+package loops.simple_programs_with_repitition.easy;
+
+/*
+Write a program that will ask the user for a sentence and then a character and 
```

Comment:
There are two problems. The first is that it is not specified in the question if we should ignore case or not, and the document code does, but yours does not. This should be corrected to avoid confusion. Also, again there is no sample output, so they can't actually match what your clause statement is testing for.

Reply:
I also just looked and saw that these are Parson questions, so technically they are not writing the code themselves, but I still feel a sample output would be useful and is something we should talk about in a meeting.